tests/optimization/bfgs.cpp

```diff
@@ -0,0 +1,149 @@
+/*
+   Copyright (c) 2009-2016, Ryan H. Lewis
+   All rights reserved.
+
+   This file is part of Elemental and is under the BSD 2-Clause License, 
+   which can be found in the LICENSE file in the root directory, or at 
+   http://opensource.org/licenses/BSD-2-Clause
+*/
+#include <utility>
+#include <El.hpp>
+#include <El/optimization/bfgs.hpp>
+#define CATCH_CONFIG_RUNNER
+#include <catch.hpp>
+
+using namespace El;
```

Comment:
I try to always use namespaces (renaming them locally when they are too long to type). it enhances readability, but, I did copy this cpp file from another one in El.